instana/instrumentation/aws/triggers.py

```diff
@@ -13,7 +13,11 @@
 
 def get_context(tracer, event):
     # TODO: Search for more types of trigger context
-    if is_api_gateway_proxy_trigger(event) or is_application_load_balancer_trigger(event):
+    is_proxy_event = is_api_gateway_proxy_trigger(event) or \
+        is_api_gateway_v2_proxy_trigger(event) or \
+        is_application_load_balancer_trigger(event)
+
+    if is_proxy_event:
         return tracer.extract('http_headers', event['headers'])
```

Comment:
The failsafe check here would be to also add in `if "headers" in event` or alternatively wrap this entire function in an try..except (or both).  What do you think?

Reply:
Fair enough, although the fact that this is a proxy event guarantees that we have `headers`, who knows when AWS decides to change their payload again. An `event.get('headers', dict())` maybe?